### ISSUE TITLE
make create_environment fails if conda activate enabled

### ISSUE THREAD

**Issue by georgerichardson** [nestauk/cookiecutter-data-science-nesta#41]
The traceback below is the last output from `make create_environment` (using [sdg-classifier](https://github.com/nestauk/sdg-classifier)). The environment is created sucesfully, but the last step of activating the environment and installing the project package fails because the local environment is set up to activate conda environments with `conda activate ...` rather than `source activate ...`.

```
source activate sdg_classifier &&  pip install -e .
/bin/sh: 1: source: not found
Makefile:91: recipe for target 'create_environment' failed
make: *** [create_environment] Error 127
```

**Comment by bishax** [comment]
So a proposed fix would be `/s/source/conda` here: https://github.com/nestauk/sdg-classifier/blob/2be9b46c44fa81923801e9ae5a3cd80744575b94/Makefile#L23 ?

**Comment by bishax** [comment]
AFAIK `source {activate,deactivate}` is deprecated and `conda {activate,deactivate}` should always work (assuming a remotely recent version of conda).

If this is not the case then `ACTIVATE_ENV := conda activate $(PROJECT_NAME) || source activate $(PROJECT_NAME)` will run `source activate ...` if `conda activate ...` fails.

**Comment by bishax** [comment]
This was closed by #42
